### ISSUE TITLE
Could not print when scale is set to a specific scale

### ISSUE THREAD

**Issue by xjin-mansfieldma** [AppGeo/GPV#138]
When try to print using a specific scale, it will fail. The other two scale options work fine. The third scale option using specific scale will generate a page saying "an error has unexpectedly occurred while starting the viewer". 
I tried the GPV app by Falmouth. The same error when printing using a specific scale.

**Comment by MikeOlkinSwsc** [comment]
The specific scale option has a placeholder value of 100. The dialog box is not actually populated when this placeholder is showing & I think that is why there are crashes. I've been able to replicate this exact issue by choosing the custom scale option, not changing the default value & clicking the Create button. Its subtle, but the placeholder text is a lighter gray than the text if a value is typed in. It's very confusing. 

We have removed the placeholder value in our GPV instance by removing the following text from the SharePanel.ascx file:
placeholder="100"
